src/Strings/PalindromeCheck.java

```diff
@@ -0,0 +1,26 @@
+package Strings;
+
+import java.util.Scanner;
+
+public class PalindromeCheck {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        //Input text
+        String stringInputted = StringUtils.stringInput(scanner);
+        //check if the word is palindrome
+        palindromeWord(stringInputted);
+
+    }
+
+    static void palindromeWord(String string)
+    {
+        boolean isPalindrome = false;
+        for (int i=0; i<string.length()/2; i++)
```

Comment:
if (yourString.equals(new StringBuilder(yourString).reverse())) --> yes, it's polidrome!

:)